api/v1alpha1/doclingserv_types.go

```diff
@@ -45,6 +45,29 @@ type DoclingServSpec struct {
 type DoclingServStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
+
+	// ObservedGeneration is the generation last observed by the controller
+	// +optional
+	DoclingObservedGeneration int64 `json:"doclingObservedGeneration,omitempty"`
```

Comment:
```suggestion
	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
```

No need to have a type on this, type is implied.